src/parser/TerraformListener.js

```diff
@@ -3,8 +3,9 @@ import antlr4 from 'antlr4';
 import {
```

Comment:
In commit `Add variable parsing` missing add test of parser with variable in resources.

Reply:
It will add too many test files in one commit, so I split in 2 commits.